site/content/en/docs/Getting started/installation.md

```diff
@@ -7,11 +7,12 @@ weight: 20
 
 ## Download and install Amazon Genomics CLI
 
-Download the Amazon Genomics CLI zip from [releases page](https://github.com/aws/amazon-genomics-cli/releases) of our Github repo.
+Download the Amazon Genomics CLI zip, unzip its contents, and run the `install.sh` script:
 
-Next, unzip its contents, and run the `install.sh` script:
+To download a specific release, see [releases page](https://github.com/aws/amazon-genomics-cli/releases) of our Github repo.
 
 ```
+wget https://github.com/aws/amazon-genomics-cli/releases/latest/download/amazon-genomics-cli.zip
```

Comment:
I like this but the only issue is that `wget` isn't available on all distros by default. Maybe change this to a `curl` command or show that as an alternative?

Reply:
Yup agreed! updated